app/src/androidTest/java/co/nimblehq/compose/crypto/ui/screen/HomeScreenUITest.kt

```diff
@@ -172,10 +172,14 @@ class HomeScreenUITest {
 
         initViewModel()
 
-        composeAndroidTestRule.onNodeWithTag(
-            testTag = TestTagCoinItem,
-            useUnmergedTree = true
-        ).assertDoesNotExist()
+        with(composeAndroidTestRule) {
+            onNodeWithTag(
+                testTag = TestTagCoinItem,
+                useUnmergedTree = true
+            ).assertDoesNotExist()
+
+            onRoot().assertExists(errorGeneric)
```

Comment:
@Wadeewee Sorry but `assertExists` will check whether the note `onRoot` exists. The param of `assertExists` is `errorMessageOnFail`, not the error to verify. 

If you change it to `onRoot().assertExists("errorGeneric")`, the test is still passed.

![image](https://user-images.githubusercontent.com/16315358/220003211-c6d7a9f8-be02-440c-b618-ec88ae79b791.png)

Reply:
Oh, Indeed, @luongvo 🙈 So I think we can't assert this for now then, Even if we can switch to `SnackBar,` but I think it better to work on another PR, What about if I leave TODO to fix it later?

Reply:
@Wadeewee I think so, I think you can drop the latest commit and replace it with TODO comments for androidTest 🙏